tasks/build_tags.py

```diff
@@ -126,7 +126,20 @@
 INSTALLER_TAGS = {"docker", "ec2", "kubelet"}
 
 # PROCESS_AGENT_TAGS lists the tags necessary to build the process-agent
-PROCESS_AGENT_TAGS = AGENT_TAGS.union({"fargateprocess"}).difference({"otlp", "python", "trivy"})
+PROCESS_AGENT_TAGS = {
```

Comment:
You might need to keep `netcgo`, this build tag controls some internal behavior of the `net` Go package (+ it should have 0 impact on binary size / memory)